src/Android/Utilities/AndroidHelpers.cs

```diff
@@ -47,5 +49,22 @@ public static async Task SetPreconfiguredRestrictionSettingsAsync(Context contex
                 await AppHelpers.SetPreconfiguredSettingsAsync(dict);
             }
         }
+
+        public static PendingIntentFlags PendingIntentMutability(PendingIntentFlags pendingIntentFlags, bool isMutable)
```

Comment:
I try to use verbs on methods, in here if you are the caller is not that obvious what the method does, you could change it to `AddPendingIntentMutabilityFlag` or something alike.